.changeset/olive-lobsters-approve.md

```diff
@@ -0,0 +1,5 @@
+---
+'@chainlink/alongside-test-adapter': minor
```

Comment:
Since this is a new EA, we should set this to a major update. We can also directly name is `alongside` and skip on the `-test` since it's not one being migrated from v2.